gremlin/ogma-neptune/README.md

```diff
@@ -0,0 +1,296 @@
+# Visualize data in Amazon Neptune using Ogma library
+
+This GitHub lab will take you through hands-on exercise of visualizing graph data in Amazon Neptune using Ogma library.
+Amazon Neptune is a fast, reliable, fully managed graph database service available from AWS.
+With Amazon Neptune you can use open source and popular graph query languages such as Apache TinkerPop Gremlin for property graph databases or SPARQL for W3C RDF model graph databases.
+
+In the rest of the lab, we will focus on visualizing data in Amazon Neptune using [Ogma](http://ogma.linkurio.us).
+Ogma is a Javascript library for large-scale interactive graph visualizations. Ogma offers set of features required to display, explore, and interact with graph data in a web application. This includes connecting to various sources to import and export data, smart layouts algorithms, rich user interactions, and fully customizable visual styles.
+
+## PREREQUISITES:
+
+1. Amazon Neptune Cluster
+2. Access to AWS IAM Permissions for
+   -> creating AWS Lambda functions
+   -> creating IAM roles for Amazon Neptune cluster to access S3 and, for API Gateway to access AWS Lambda functions
+   -> APIs in Amazon API Gateway
+   -> Amazon S3 buckets
+   -> creating and attaching VPC Endpoints to the VPC
+3. Ogma library
+
+## Architecture
+
+Below is the architecture that we will be using to visualize data in Amazon Neptune using [Ogma](https://ogma.linkurio.us).
+Since Amazon Neptune instance can't be accessed outside the VPC, we are using AWS Lambda function in VPC. This AWS Lambda function is accessed through the proxy created and exposed to internet using Amazon API Gateway.
+Once the proxy is exposed, we can access the APIs from Javascript code being executed from Amazon S3 (static website).
+
+![](images/architecture.png)
+
+## Steps
+
+1. [Provision Amazon Neptune Cluster](#1-provision-amazon-neptune-cluster)
+2. [Load sample data into Amazon Neptune](#2-load-sample-data-into-amazon-neptune)
+3. [Create and Configure AWS Lambda Function](#3-create-and-configure-aws-lambda-function)
+4. [Create and Configure Amazon API Gateway - Proxy API](#4-create-and-configure-amazon-api-gateway---proxy-api)
+5. [Configure Amazon S3 bucket for hosting a static website](#5-configure-amazon-s3-bucket-for-hosting-a-static-website)
```

Comment:
Change to "Configure the hosting for the static website" as per the new heading.